spec/request/service_credential_bindings_spec.rb

```diff
@@ -1868,13 +1873,19 @@ def check_filtered_bindings(*bindings)
       end
 
       context 'app binding' do
-        it_behaves_like 'service credential binding delete endpoint', 'service_binding', VCAP::CloudController::ServiceBinding
+        it_behaves_like 'service credential binding delete endpoint', 'service_binding',
```

Comment:
I have a preference for `'service_binding'` to be on it's own line as it highlights that it's an argument like the others